index.html

```diff
@@ -1,13 +1,246 @@
+          <ul class="project-maker">
+            <li class="project-owner">Adarsh</li>
+            <li>future fullstack</li>
+            <li>2022</li>
+          </ul>
+          <p class="description info">
+            I made this panguin css-animation when i was learning css month ago with free code camp
+            and i love this project do checkout
+          </p>
+          <ul class="tech">
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <a class=buttonlink href="https://codepen.io/PowerLevel9000/pen/mdKGWej" target="_blank" rel="noopener noreferrer"><button>See Project</button></a>
+        </div>
+      </div>
+
+    </section>
+
+    <section class="aboutme">
+      <h2>About <br> Myself</h2>
+      <p class="description info2" style="padding-left: 20px;">
+        Hello I&apos;m a software developer! I can help you 
+        build a product, feature or website Look through some of my work and 
+        experience! If you like what you see and have a project you need coded, 
+        don&apos;t hestiate to contact me.
+      </p>
+      
+      <h3 class="connect"><a href="mailto:adi8090808766@gmail.com" target="_blank"></a>Let&apos;S CONNECT</h3>
+      
+      <ul class="social">
+        <li>
+          <a href="https://github.com/PowerLevel9000" target="_blank">
+            <i class="fa-brands fa-github"></i>
+          </a>
+        </li>
+
+        <li>
+          <a href="https://instagram.com/adarsh8090adi?igshid=YmMyMTA2M2Y=" target="_blank">
+            <i class="fa-brands fa-instagram"></i>
+          </a>
+        </li>
+
+        <li>
+          <a href="https://www.facebook.com/profile.php?id=100010279544328" target="_blank">
+            <i class="fa-brands fa-facebook"></i>
+          </a>
+        </li>
+
+        <li>
+          <a href="https://youtube.com/@code_gamer2.068" target="_blank">
+            <i class="fa-brands fa-youtube"></i>
+          </a>
+        </li>
+
+      </ul>
+
+      <button>my resume</button>
+      
+      <ul class="skills">
+        <li class="lang">Languages <span class="right"><img src="media/caret-down-solid.svg" alt="dropdown" style="display: inline-block; width:20px; height:20px"></span>
+       
+          <ul class="languages">
+            <li><img src="media/html.png" alt="html logo"><span>Html</span></li>
+            <li><img src="media/css.png" alt="css logo"><span>Css</span></li>
+            <li><img src="media/javascript.png" alt="javascript logo"><span>JavaScript</span></li>
+          </ul>
+        </li> 
+        <div class="border"></div>
+        <li>framework <span class="right"><img src="media/caret-down-solid.svg" alt="dropdown" style="display: inline-block; width:20px; height:20px"></span></li>
+        <li>skills <span class="right"><img src="media/caret-down-solid.svg" alt="dropdown" style="display: inline-block; width:20px; height:20px"></span></li>
+      </ul>
+
+
+    </section>
```

Comment:
<img width="669" alt="s5" src="https://user-images.githubusercontent.com/77843050/205440113-66bf5189-379b-49bb-9ccb-af379221cb93.png">

- [ ] The text `my resume` is not the same as `get my resume` on the Figma design, please change that.
- [ ] The arrow icons are not the same as the one on the original design, please make sure you update that one too.
- [ ] Take a look closer at the language container, it has a smooth gray background, right? but we do not see it on your design page, please add that as mentioned in the image above.
- [ ] Same as the lines separating the frameworks and the languages, yours are much black than those on the Figma, please touch it up.